docs/project-setup.md

```diff
@@ -0,0 +1,79 @@
+# Setting up a new project
+This document covers the basic project management tools and procedures recommended for Special Interest Group developing an implementation of the OpenTelemetry Specification for a given programming language.
+
+A Special Interest Group focused on implemeting OpenTelemetry in a programming language is reffered to as a **project** for the remainder of this document.
+
+If you are interested in starting a new project, please do the following
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new project.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+
+# Roadmap to Convergence
+For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the initial goals and timelines for OpenTelemetry.
+
+## OpenTelemetry Timeline
+**4/24** The reference candidate is presented for review.
+
+**5/8** Teams form, and work begins in all languages.
+
+**9/6** C#, Golang, Java, NodeJS, and Python reach parity with existing projects.
```

Comment:
this milestone is the only one to add then?

Reply:
Can we also mention Erlang here?

Reply:
@deadtrickster as long as we have you committed for it =). The thinking was that we have some people committed for those languages and we do not want to over-promise

Reply:
Yes I'm ;-) and I'm not alone. We have @tsloughter and the whole Observability WG: https://erlef.org/observability-wg/

Reply:
I ended up removing this section anyways, so it's a not issue.

Related to creating a common set of milestones: I want to track SIG progress on a big board of some kind, preferably on the website. So it will be easy to see overall progress. I look forwards to seeing Erlang in the lead! :)